tensorflow/examples/udacity/1_notmnist.ipynb

```diff
@@ -183,15 +183,17 @@
       "source": [
         "num_classes = 10\n",
         "\n",
-        "def extract(filename):\n",
-        "  tar = tarfile.open(filename)\n",
+        "def maybe_extract(filename):\n",
```

Comment:
No, I don't want that behavior. If someone accidentally cancels the decompression in the middle, they have no way to recover cleanly.
If you don't want to re-decompress, simply don't rerun this frame.
Note that the situation is different for the download since we want to avoid DDOSing the server hosting the data, and we have a check for the integrity of the downloaded file to warn if the file is only half downloaded.